FIXME: ugly hack to find conf.pri with vpath qmake

--- origsrc/esperanza/config.pri	2010-05-24 14:37:14.000000000 -0500
+++ src/esperanza/config.pri	2010-05-24 14:50:10.526181300 -0500
@@ -11,7 +11,8 @@ macx {
 }
 
 !win32 {
-	include (conf.pri)
+	include (../../build/conf.pri)
+	LIBS += $$QMAKE_LIBS_X11
 }
 
 win32 {
--- origsrc/esperanza/src/ui/ui.pro	2010-05-24 14:37:15.000000000 -0500
+++ src/esperanza/src/ui/ui.pro	2010-05-24 14:42:25.590588500 -0500
@@ -22,7 +22,7 @@ win32 {
 	COMPONENTS+=../lib/release/liblib.a ../../data/release/libdata.a
 }
 
-LIBS += $$COMPONENTS
+LIBS = $$COMPONENTS $$LIBS
 
 PRE_TARGETDEPS = $$COMPONENTS
 
